src/test/java/net/smartcosmos/dao/relationships/converter/RelationshipCreateToRelationshipEntityConverterTest.java

```diff
@@ -0,0 +1,54 @@
+package net.smartcosmos.dao.relationships.converter;
+
+import org.junit.*;
+
+import net.smartcosmos.dao.relationships.domain.RelationshipEntity;
+import net.smartcosmos.dao.relationships.util.UuidUtil;
+import net.smartcosmos.dto.relationships.RelationshipCreate;
+import net.smartcosmos.dto.relationships.RelationshipReference;
+
+import static org.junit.Assert.*;
+
+/**
+ * RelationshipCreateToRelationshipEntityConverter Tester.
+ *
+ * @author tcross
+ * @version 1.0
+ * @since <pre>Jun 28, 2016</pre>
+ */
+public class RelationshipCreateToRelationshipEntityConverterTest {
+
+    @Before
+    public void before() throws Exception {
+    }
+
+    @After
+    public void after() throws Exception {
+    }
+
+    /**
+     * Method: convert(RelationshipCreate createRelationship)
+     */
+    @Test
+    public void testConvert() throws Exception {
+        final String TEST_SOURCE_URN = "urn:thing:uuid:" + UuidUtil.getNewUuidAsString();
+        final String TEST_TARGET_URN = "urn:thing:uuid:" + UuidUtil.getNewUuidAsString();
+        final String TEST_SOURCE_TYPE = "Thing";
```

Comment:
Always using `Thing` as `type` might be misleading going forward, because the value actually is the Thing type (in case of Things).